Documentation/Developer/Index.rst

```diff
@@ -0,0 +1,38 @@
+.. include:: ../Includes.txt
+
+
+.. _developer:
+
+================
+Developer Corner
+================
+
+Build your own widget
+=====================
+
+.. todo::
+
+   define how developers can build their own widgets
+
+-------------------------
```

Comment:
This line is not needed. See https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/CheatSheet.html